website/blog/2023-03-23-5-things-to-know-for-a-docker-user.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 5 things to know about Podman Desktop for a Docker user
+description: Important things to know when switching from Docker Desktop to Podman Desktop
+slug: 5-things-to-know-for-a-docker-user
+authors: [benoitf]
+tags: [podman-desktop, docker, migrating]
+hide_table_of_contents: false
+---
+
+The 5 things to know being a Docker user by using Podman Desktop:
+
+- **Use a single UI**: Podman Desktop works with several container engines, including Docker.
+- The **compatibility mode**: How to ensure tools are working with Podman instead of Docker.
+- **Compose** support. How to work with Compose files and Podman.
+- **Kubernetes**. How to use Kubernetes with Podman.
+- **rootless** mode or containers without root privileges.
```

Comment:
```suggestion
- **Secutiry**: Use `rootless` mode or containers without root privileges.
```

Reply:
looks like there is a typo in your suggested change